matching-notification-rest/src/main/java/org/phenotips/matchingnotification/rest/MatchingNotificationResource.java

```diff
@@ -226,4 +226,17 @@ Response setUserContacted(@FormParam("matchesIds") Set<Long> matchesIds,
     @Produces(MediaType.APPLICATION_JSON)
     @Path("/save-comment")
     Response saveComment(@FormParam("matchesIds") Set<Long> matchesIds, @FormParam("comment") String comment);
+
+    /**
+     * Saves note for matches, with ids given in parameter.
```

Comment:
I would rename all public `saveNote()` methods to `addNote()`, to clearly indicate that notes are never edited or replaced, and only new ones are added (by design) - would add this to the comments as well

Reply:
renamed